src/components/DropDownMenu/DropDownMenu.tsx

```diff
@@ -0,0 +1,43 @@
+import cn from 'classnames';
+import { Person } from '../../types/Person';
+
+type Props = {
+  personList: Person[]
+  selectPerson: (person: Person) => void;
+};
+
+export const DropDownMenu: React.FC<Props> = ({ personList, selectPerson }) => {
+  return (
+    <div className="dropdown-menu" role="menu">
+      <div className="dropdown-content">
+        {personList.length !== 0
+          ? personList.map(person => (
+            <a
+              href="/#"
```

Comment:
```suggestion
              href="/"
```
the issue, that you've mentioned is because on click you should also set query (add the name of the person to a browser route)